cmd/harmony/main.go

```diff
@@ -480,11 +480,9 @@ func main() {
 	go currentNode.SupportSyncing()
 	currentNode.ServiceManagerSetup()
 
-	// RPC for SDK not supported for mainnet.
-	if *networkType != nodeconfig.Mainnet {
```

Comment:
Why remove this condition?

Reply:
because from now we want to enable the sdk by default